input/resources/au-erequesting-placer.xml

```diff
@@ -83,5 +83,252 @@
         </extension>
       </referencePolicy> -->
     </resource>
+    <!-- COVERAGE -->
```

Comment:
Not the line that needs to be fixed but commenting here to get an inline comment. 

In rest.description, please replace 'Server' with 'Placer".

Reply:
fixed.